templates/person/detail.html

```diff
@@ -183,7 +183,8 @@ <h3 id="subordinates"><i class="fa fa-fw fa-share-alt fa-rotate-90"></i> {% tran
             return new_list
         }; 
 
-        $(document).ready(function() {
+        $(window).load(function () {
```

Comment:
What's your thinking here? Is it for the JS cursor changes below?

Reply:
Oh, right...I can change that back. I was trying out various solutions for the Flickity vs. vis.js battle royale. This accomplished nothing.

Reply:
Ahhh OK got it!